### ISSUE TITLE
ora2pg does not convert anydata

### ISSUE THREAD

**Issue by did16** [darold/ora2pg#1642]
Ora2Pg v23.2 does not convert Oracle Anydata type to PostgreSQL:
--- Oracle
CREATE TABLE "CTR"."TQ_ACTIV" 
   (	... 
"USER_PROP" "SYS"."ANYDATA" , 
...
--- PostgreSQL 
CREATE TABLE tq_activ (
...
	user_prop ANYDATA

anydata is not a postgresql type

**Comment by darold** [comment]
Right, converting this depend of the use but such all-in-one data type do not exists in PostgreSQL. Some are converting it to json but my opinion is that an extension should be developed to emulate this datatype. The other solution is to modify the application logic and design to avoid the use of such insane column.
